profileCC breaks when there is no rating for a user in a specific chess category like bullet

**Describe the bug**
profileCC breaks when there is no rating for a user in a specific chess category like bullet

**To Reproduce**
Steps to reproduce the behavior:
1. /profilecc cnunezre

**Expected behavior**
I would expect to show the category with a default value of zero, empty or ? like in the  lichess command /profile. 

**Screenshots**
<img width="948" alt="Screenshot 2024-06-19 at 4 00 25 PM" src="https://github.com/jalpp/Chesslise/assets/1489384/c35eacce-5d10-4f70-8737-64576322923b">

Exception when testing locally
`2024-06-19 15:56:24 [JDA] [ERROR] One of the EventListeners had an uncaught exception
java.lang.NullPointerException: Cannot invoke "io.github.sornerol.chess.pubapi.domain.player.stats.RatingPoolStats.getLast()" because the return value of "io.github.sornerol.chess.pubapi.domain.player.stats.PlayerStats.getChessBullet()" is null
	at Chesscom.CCProfile.getCCProfile(CCProfile.java:27)
	at Discord.HelperModules.ModalHelperContextModule.handleChessComProfileOnFormSubmit(ModalHelperContextModule.java:114)
	at Discord.HandlerModules.ModalContextModule.handleLogic(ModalContextModule.java:25)
	at Discord.MainHandler.CommandHandler.onModalInteraction(CommandHandler.java:48)
	at net.dv8tion.jda.api.hooks.ListenerAdapter.onEvent(ListenerAdapter.java:463)
	at net.dv8tion.jda.api.hooks.InterfacedEventManager.handle(InterfacedEventManager.java:98)
	at net.dv8tion.jda.internal.hooks.EventManagerProxy.handleInternally(EventManagerProxy.java:88)
	at net.dv8tion.jda.internal.hooks.EventManagerProxy.handle(EventManagerProxy.java:70)
	at net.dv8tion.jda.internal.JDAImpl.handleEvent(JDAImpl.java:169)
	at net.dv8tion.jda.internal.handle.InteractionCreateHandler.handleInternally(InteractionCreateHandler.java:94)
	at net.dv8tion.jda.internal.handle.SocketHandler.handle(SocketHandler.java:39)
	at net.dv8tion.jda.internal.requests.WebSocketClient.onDispatch(WebSocketClient.java:1009)
	at net.dv8tion.jda.internal.requests.WebSocketClient.onEvent(WebSocketClient.java:892)
	at net.dv8tion.jda.internal.requests.WebSocketClient.handleEvent(WebSocketClient.java:870)
	at net.dv8tion.jda.internal.requests.WebSocketClient.onBinaryMessage(WebSocketClient.java:1048)
	at com.neovisionaries.ws.client.ListenerManager.callOnBinaryMessage(ListenerManager.java:385)
	at com.neovisionaries.ws.client.ReadingThread.callOnBinaryMessage(ReadingThread.java:276)
	at com.neovisionaries.ws.client.ReadingThread.handleBinaryFrame(ReadingThread.java:996)
	at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:755)
	at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:108)
	at com.neovisionaries.ws.client.ReadingThread.runMain(ReadingThread.java:64)
	at com.neovisionaries.ws.client.WebSocketThread.run(WebSocketThread.java:45)
`

--- a/src/main/java/Chesscom/CCProfile.java
+++ b/src/main/java/Chesscom/CCProfile.java
@@ -2,6 +2,8 @@ package Chesscom;
 
 import Abstraction.Player.UserObject;
 import io.github.sornerol.chess.pubapi.domain.player.stats.PlayerStats;
+import io.github.sornerol.chess.pubapi.domain.player.stats.PuzzleRushStats;
+import io.github.sornerol.chess.pubapi.domain.player.stats.RatingPoolStats;
 import io.github.sornerol.chess.pubapi.exception.ChessComPubApiException;
 import net.dv8tion.jda.api.EmbedBuilder;
 
@@ -24,13 +26,13 @@ public class CCProfile extends UserObject {
             PlayerStats player = this.getChessComStats();
 
             proSay += "**Bullet**: " +
-                    player.getChessBullet().getLast().getRating() +
+                    getRatingOrDefault(player.getChessBullet()) +
                     "\n **Rapid:**  " +
-                    player.getChessRapid().getLast().getRating() +
+                    getRatingOrDefault(player.getChessRapid()) +
                     "\n **Blitz:** " +
-                    player.getChessBlitz().getLast().getRating() +
+                    getRatingOrDefault(player.getChessBlitz()) +
                     "\n **PuzzleRush:** " +
-                    player.getPuzzleRush().getBest().getScore();
+                    getStatsOrDefault(player.getPuzzleRush());
             embedBuilder.setThumbnail(this.getPlayerClient().getPlayerByUsername(getUserID()).getAvatarUrl()).setTitle(this.getUserID() + "'s Chess.com Profile").setDescription(proSay).setColor(Color.green);
 
         } catch (IOException e) {
@@ -40,8 +42,13 @@ public class CCProfile extends UserObject {
         }
 
         return embedBuilder;
-
     }
 
+    private String getRatingOrDefault(RatingPoolStats rating) {
+        return rating != null ? rating.getLast().getRating().toString() : "?";
+    }
 
+    private String getStatsOrDefault(PuzzleRushStats stats) {
+        return stats != null ? stats.getBest().getScore().toString() : "?";
+    }
 }

